.travis.yml

```diff
@@ -8,11 +8,18 @@ python:
  - "3.4"
  - "3.5"
  - "pypy"
+ - "pypy3"
+ - "pypy3.3-5.2-alpha1"
  - "nightly"
-install: pip install .
-script: tox
-branches:
-  only:
-    - master
-matrix:
-  fast_finish: true
+
+before_install:
+ # Show the current setuptools version
+ - ORIG_SETUPTOOLS_VERSION=`python -c "import setuptools; print('%s' % setuptools.__version__)"`
+ - echo setuptools $ORIG_SETUPTOOLS_VERSION
+install:
+ - pip install wheel
+ - python setup.py install bdist_wheel
+ - pip install ./dist/tox_travis-*.whl
```

Comment:
`pip install ..` here is making sure the wheel is installable by pip.
If it isnt installable by pip, tox is going to fail in the most ugly of ways.